.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!--
+    If you are changing an existing module or resource, describe why that configuration is no longer acceptable. If you are fixing a misconfiguration, make sure to describe both the intended result and the actual (faulty) result.
+-->
+
+<!--
+    If you are adding a module or resources, describe what usage will be supported when this change is applied.
+-->
+
+<!--
+    Explain what alternative configurations you have evaluated and why the chosen configuration is preferable.
+-->
+
+<!--
+    What do reviewers need to know to review? Link pull requests or issues that are related.
+-->
+
+<!--
+    What do we need to consider when rolling out this change?
+-->
+
+Fixes #NNN <!-- This will close the issue automatically when you merge -->
+
+Completion workflow:
+
+- [ ] Ensure all pull request comments are addressed
+- [ ] Rebase your branch against main branch
+- [ ] Ensure that all checks pass with the latest changes on main branch
+- Merge the pull request.
```

Comment:
Different way of doing things but I like the changes in a PR to be squashed unless there are commits that need to be in the history separately (not that common)

Reply:
I think I do not understand the comment. Is there anything in the list that should be different? I'm deliberately not saying anything about how you craft the commits because it varies with taste and task at hand, so long as you use merge rather than fast-forward.

Reply:
Ok, I did the wrong interpretation here. I was thinking of the different PR options “Merge” and “Squash and merge”